EdFi.Ods.Utilities.Migration/Scripts/PgSql/02Upgrade/v51_to_v52/01 Structure/1010-AddNewTablesForV52.sql

```diff
@@ -0,0 +1,245 @@
+    CONSTRAINT fk_a20588_assessment FOREIGN KEY (AssessmentIdentifier, Namespace) REFERENCES edfi.Assessment (AssessmentIdentifier, Namespace),
+    CONSTRAINT fk_a20588_assessmentreportingmethoddescriptor FOREIGN KEY (AssessmentReportingMethodDescriptorId) REFERENCES edfi.AssessmentReportingMethodDescriptor (AssessmentReportingMethodDescriptorId),
+    CONSTRAINT fk_a20588_objectiveassessment FOREIGN KEY (AssessmentIdentifier, IdentificationCode, Namespace) REFERENCES edfi.ObjectiveAssessment (AssessmentIdentifier, IdentificationCode, Namespace)
+);
+
+CREATE INDEX fk_a20588_assessment
+    ON edfi.AssessmentScoreRangeLearningStandard(AssessmentIdentifier , Namespace);
+
+CREATE  INDEX fk_a20588_assessmentreportingmethoddescriptor
+    ON edfi.AssessmentScoreRangeLearningStandard(AssessmentReportingMethodDescriptorId );
+
+CREATE  INDEX fk_a20588_objectiveassessment
+    ON edfi.AssessmentScoreRangeLearningStandard(AssessmentIdentifier , IdentificationCode , Namespace );
+
+CREATE UNIQUE INDEX ux_a20588_id
+    ON edfi.AssessmentScoreRangeLearningStandard(Id );
+
+CREATE TABLE edfi.AssessmentScoreRangeLearningStandardLearningStandard (
+    AssessmentIdentifier VARCHAR (60)  NOT NULL,
+    LearningStandardId   VARCHAR (60)  NOT NULL,
+    Namespace            VARCHAR (255) NOT NULL,
+    ScoreRangeId         VARCHAR (60)  NOT NULL,
+    CreateDate           TIMESTAMP   DEFAULT current_timestamp NOT NULL,
+    CONSTRAINT AssessmentScoreRangeLearningStandardLearningStandard_PK PRIMARY KEY  (AssessmentIdentifier , LearningStandardId , Namespace , ScoreRangeId ),
+    CONSTRAINT fk_9960a9_assessmentscorerangelearningstandard FOREIGN KEY (AssessmentIdentifier, Namespace, ScoreRangeId) REFERENCES edfi.AssessmentScoreRangeLearningStandard (AssessmentIdentifier, Namespace, ScoreRangeId) ON DELETE CASCADE,
+    CONSTRAINT fk_9960a9_learningstandard FOREIGN KEY (LearningStandardId) REFERENCES edfi.LearningStandard (LearningStandardId)
+);
+
+
+CREATE  INDEX fk_9960a9_assessmentscorerangelearningstandard
+    ON edfi.AssessmentScoreRangeLearningStandardLearningStandard(AssessmentIdentifier , Namespace , ScoreRangeId );
+
+
+CREATE  INDEX fk_9960a9_learningstandard
+    ON edfi.AssessmentScoreRangeLearningStandardLearningStandard(LearningStandardId );
+
+
+
+CREATE TABLE edfi.StudentDisciplineIncidentBehaviorAssociation (
+    BehaviorDescriptorId        INT              NOT NULL,
+    IncidentIdentifier          VARCHAR (20)    NOT NULL,
+    SchoolId                    INT              NOT NULL,
+    StudentUSI                  INT              NOT NULL,
+    BehaviorDetailedDescription VARCHAR (1024)  NULL,
+    Discriminator               VARCHAR (128)   NULL,
+    CreateDate                  TIMESTAMP    DEFAULT current_timestamp NOT NULL,
+    LastModifiedDate            TIMESTAMP    DEFAULT current_timestamp NOT NULL,
+    Id                          UUID DEFAULT gen_random_uuid() NOT NULL,
```

Comment:
In ODS for v5.2 from the package, this line seems like it should end as `UUID  DEFAULT public.gen_random_uuid() NOT NULL,`

Reply:
fixed it